src/main/java/uk/gov/justice/probation/courtcaseservice/restclient/prisonapi/PrisonApiSentenceDetail.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.justice.probation.courtcaseservice.restclient.prisonapi;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.time.ZonedDateTime;
+
+@Getter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor(access = AccessLevel.PRIVATE, force = true)
+public class PrisonApiSentenceDetail {
+    private final ZonedDateTime homeDetentionCurfewActualDate;
```

Comment:
JUst a matter of interest, why ZonedDateTime ? Is this just to match what the prison API are doing ? If so, why do they do this ?

Reply:
Yeah if you look at the docs [here](https://api-dev.prison.service.justice.gov.uk/swagger-ui/#/offenders/getOffenderSentenceDetails) you can see they include time and offset in the response but I'm told it's never used and only ever treated as a date. As to why - historical reasons! I think it was Andy M who told me that 🤔